timed/tracking/serializers.py

```diff
@@ -22,56 +21,30 @@
 class ActivitySerializer(ModelSerializer):
     """Activity serializer."""
 
-    duration = DurationField(read_only=True)
     user     = CurrentUserResourceRelatedField()
     task     = ResourceRelatedField(queryset=Task.objects.all(),
```

Comment:
as we are working on this code bit... this definition of task is actually not needed as this will be the default. This can be removed and serializer will do it automatically.

Reply:
Done